packages/cli/src/commands/daemon/start.ts

```diff
@@ -32,6 +34,11 @@ export default class Start extends Command {
       default: 26656,
       required: true,
     }),
+    network: flags.string({
+      description: 'Name of the network to connect to',
+      default: 'mesg-dev-chain',
```

Comment:
let's make this argument optional with empty string as default

Reply:
done